<launch>
	<node name="human_pose" pkg="human_motions" type="human_model" output="screen">
		<param name="odom_topic" value="/bill" />
		<param name="path_size" value="40" />
		<param name="predict_time" value="5.0" />
	</node>
	<node name="bag_player" pkg="rosbag" type="play" output="screen" args="-l $(find human_motions)/data/traj_ros.bag">
	</node>
	<node name="rviz" pkg="rviz" type="rviz" output="screen">
	</node>
</launch>
